data/json/items/tools.json

```diff
@@ -5610,7 +5610,31 @@
     "category" : "weapons",
     "use_action": { "type": "knife" }
   },
-
+  {
+    "id": "glaive",
```

Comment:
Well, it's odd to have makeshift glaive usable as a knife, but have a proper glave unusable in such a way. Should I remove ability to use makeshift glaive as a tool then?